smoke-test/src/test/java/org/opennms/smoketest/BSMAdminIT.java

```diff
@@ -116,6 +128,24 @@ public BsmAdminPageEditWindow addReductionKeyEdge(String reductionKeyText, Strin
             return this;
         }
 
+        public BsmAdminPageEditWindow addAttribute(String key, String value) throws InterruptedException {
+            newAttributeWindow()
+                    .key(key)
+                    .value(value)
+                    .confirm();
+            wait.until(ExpectedConditions.elementToBeClickable(By.id("addAttributeButton")));
+            return this;
+        }
+
+        public BsmAdminPageEditWindow editAttribute(String key, String value) throws InterruptedException {
+            getSelectWebElement("attributeList").selectByVisibleText(key);
+            editAttributeWindow()
+                    .value(value)
+                    .confirm();
+            wait.until(ExpectedConditions.elementToBeClickable(By.id("addAttributeButton")));
```

Comment:
This is kinda a duplicate, as we already wait in the confirm() method or is there a particular reason to wait again for another element?